app/util/StreamAudio.js

```diff
@@ -57,5 +57,9 @@ SDL.StreamAudio = {
       this.audio.src = '';
       this.audio.pause();
     }
+  },
+
+  toggleMute: function() {
+    this.audio.muted = !this.audio.muted;
```

Comment:
```suggestion
  setMuted: function(muted) {
    this.audio.muted = muted;
```

Probably better to explicitly set this value, just to avoid running into issues where the state gets out of phase

Reply:
Hi @jacobkeeler! Thank you for you comment! Please look at changes I've added in [commit](https://github.com/smartdevicelink/sdl_hmi/pull/655/commits/94af0c8fb50dbb6be4195e064316af787f63a9d3)